itou/companies/migrations/0009_remove_company_rdv_insertion_id.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.0.9 on 2024-09-12 14:43
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("companies", "0008_migrate_company_rdv_insertion_id_to_rdv_solidarites_id"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.RemoveField(
+                    model_name="company",
+                    name="rdv_insertion_id",
+                ),
+            ],
+            # Don't drop anything yet, since the previous will still be running when this migration will be run.
+            # Keeping this field around should be an issue since it is NULLABLE.
```

Comment:
```suggestion
            # Keeping this field around should not be an issue since it is NULLABLE.
```
(oui ça ressemble à un commentaire que j'aurais pu écrire :grimacing: )